docs/examples/models_signature.py

```diff
@@ -0,0 +1,9 @@
+import inspect
+from pydantic import BaseModel
+
+class FooModel(BaseModel):
+    id: int
+    name: str = None
+    description: str = 'Foo'
```

Comment:
```suggestion
    description: str = 'Foo'
    apple: int = Field(..., alias='pear')
```
(or something more sensible) good to include an example of an alias.